diploma-project/components/DropFileZone.jsx

```diff
@@ -1,18 +1,70 @@
+import {useState} from "react";
+
 const DropFileZone = ({isUploading, onImageChange}) => {
+    const [isDragging, setIsDragging] = useState(false);
+
+    const handleDragOver = (event) => {
+        event.preventDefault();
+        setIsDragging(true);
+    };
+
+    const handleDragLeave = () => {
+        setIsDragging(false);
+    };
+
+    const handleDrop = (event) => {
+        event.preventDefault();
+        setIsDragging(false);
+
+        const file = event.dataTransfer.files[0];
```

Comment:
The handleDrop and handleChange functions currently accept any type of file. Depending on your specific use case, you might want to add some checks to ensure that the dropped or selected file is of the correct type (e.g., an image). Although you have specified the accept attribute in the input field, it's still a good practice to validate the file type in your JavaScript code.

Reply:
Its okay if I make a basic validation of the file extension in my JavaScript code, right?

Reply:
Yep, that should be enough